style.css

```diff
@@ -425,3 +432,234 @@ footer {
 .footer_social_icons li a img {
   height: 20px;
 }
+
+@media (min-width: 768px) {
+  nav {
+    padding: 26px 151px;
+  }
+
+  .logo {
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop_nav {
+    display: flex;
+  }
+
+  .desktop_nav li {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    list-style: none;
+    color: #fff;
+    margin-left: 32px;
+  }
+
+  .multi_stories img {
+    width: 100%;
+  }
+
+  .desktop_nav li a img {
+    height: 18px;
+  }
+
+  .header_content {
+    padding-left: 143px;
+    position: relative;
+    background-image: url('https://res.cloudinary.com/david79/image/upload/v1674327353/work%20stuff/deep%20color%20icons/Illustration-header_rvf9xd.png');
+    background-repeat: no-repeat;
+    background-size: 40%;
+    background-position-x: right;
+    background-position-y: -20px;
+  }
```

Comment:
- [ ] While you did a wonderful job styling the first section of your project :100: I think it would be better if you align the presentation text to the middle to match as much as possible with the Figma template :+1: 

**Your design**

![Screenshot from 2023-01-26 07-52-04](https://user-images.githubusercontent.com/105117832/214842941-0ddc170b-c34b-4a5e-9fe1-abc662e2cf50.png)

**Figma design**

![Screenshot from 2023-01-26 07-51-36](https://user-images.githubusercontent.com/105117832/214842988-e1885d66-f45b-4349-9da0-ed4c0a35e976.png)

Reply:
Hi @IngDavidTM 
Please can you expand on this because I can't see the correction you said I should make?

Reply:
Hi @dara-soji, what I mean is that your text should be aligned to the center, just like the Figma Template

This is your design
![Screenshot from 2023-01-26 09-29-13](https://user-images.githubusercontent.com/105117832/214862091-77c48a79-0c6f-4e64-83a1-bdb69d2ae597.png)

This is the requirement 
![Screenshot from 2023-01-26 09-31-57](https://user-images.githubusercontent.com/105117832/214862392-f7b11dc5-e351-43d4-b3bc-173ce0c71eff.png)